src/project.c

```diff
@@ -396,7 +396,7 @@ void freedata(Project *pr)
     // Free memory for node data
     if (pr->network.Node != NULL)
     {
-        for (j = 1; j <= pr->network.Nnodes; j++)
+        for (j = 1; j <= pr->parser.MaxNodes; j++)
```

Comment:
are you intentionally reverting this?

Reply:
No. My updates to a local copy of project.c were made yesterday before the last merge into dev. Then today, after creating a new branch from dev I simply copied that version of project.c into it thus overwriting the change pushed yesterday. My bad -- I will fix it shortly.